### PR TITLE
tests/utils: a very minor cleanup

### PR DESCRIPTION
**Pull request by dankenigsberg**
Make test/utils.go slightly shorter by moving `createArchive` into its only user.

```release-note
NONE
```

### CODE REVIEW

File: tests/storage/imageupload.go [hunk 1]
```diff
@@ -39,6 +39,17 @@ const (
 	insecure             = "--insecure"
 )
 
+func createArchive(targetFile, tgtDir string, sourceFilesNames ...string) string {
```
Comment (xpivarc) [inline tests/storage/imageupload.go:42]: We could put it closer to usage. WDYT?

**Review by xpivarc — COMMENTED** [review]
/approve

File: tests/storage/imageupload.go (hunk 1 above)
Comment (dankenigsberg) [reply tests/storage/imageupload.go]: You are very much correct - this function is a minor implementation detail; it does not deserve prime location at the top of the file.

**Pull request merged**